store/tikv/metrics.go

```diff
@@ -69,15 +69,6 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(0.0005, 2, 20),
 		})
 
-	connPoolHistogram = prometheus.NewHistogramVec(
```

Comment:
This is removed? why?

Reply:
this is useless after we use grpc instead of tcp, dead code for long time.